mlflow/utils/search_utils.py

```diff
@@ -363,11 +366,20 @@ def _validate_order_by_and_generate_token(cls, order_by):
     def _parse_order_by_string(cls, order_by):
         token_value = cls._validate_order_by_and_generate_token(order_by)
         is_ascending = True
-        if token_value.lower().endswith(" desc"):
-            is_ascending = False
-            token_value = token_value[0:-len(" desc")]
-        elif token_value.lower().endswith(" asc"):
-            token_value = token_value[0:-len(" asc")]
+        tokens = token_value.split()
+        if len(tokens) > 2:
+            raise MlflowException(f"Invalid order_by clause '{order_by}'. Could not be parsed.",
+                                  error_code=INVALID_PARAMETER_VALUE)
+        elif len(tokens) == 2:
+            order_token = tokens[1].lower()
+            if order_token not in cls.VALID_ORDER_BY_TAGS:
+                raise MlflowException(f"Invalid ordering key in order_by clause '{order_by}'.",
```

Comment:
this does cause a behavior change for `search_runs` (so this is technically a breaking change), right? i see that lines 355-362 above check the ordering keywords in some way, so just wanted to confirm.

Reply:
Yeah that's true - it is breaking